src/main/webapp/index.html

```diff
@@ -22,7 +22,14 @@ <h1>
           Discover the world's hidden gems <br />
           before planning your next trip.
         </h1>
-        <input type="text" name="search" placeholder="Search a city, country, etc." />
+        
+        <!-- Search Bar -->
+        <form autocomplete="off">
+          <div class="autocomplete">
+            <input id="myInput" type="text" name="Country" placeholder="Search a country" />
```

Comment:
The search results page [expects](https://github.com/gferioli0418/travelbud/blob/master/src/main/webapp/SearchResults.js#L28) a query parameter with name `q` so I think this should be:

```suggestion
            <input id="myInput" type="text" name="q" placeholder="Search a country" />
```

In any case, once you have the code to redirect, you can check the URL in your address bar to confirm what query parameters are being generated.

Reply:
I searched "United Kingdom" in the search bar and this is the URL it brought me to:
![image](https://user-images.githubusercontent.com/60808509/80039501-f1e59480-84c5-11ea-81b0-576787af3828.png)

I think the query parameters look right? However, I am not being redirected to `/SearchResults.html`.

Reply:
I think the issue is because we're using a form here instead of a proper button.